### ISSUE TITLE
Django Admin - Table improvements

### ISSUE THREAD

**Issue by reitermb** [raft-tech/TANF-app#1410]
**Description:**
- As a django admin user I want summary data to be presented via the table caption attribute in tables I interact with
- As a django admin user using a screen reader I want a more explicit way to select all rows in a table

**Acceptance Criteria:** 
- [ ] All data tables in django use table captions to hint/get read sort instructions for columns (ex: https://adrianroselli.com/2021/04/sortable-table-columns.html#Caption)
- [ ] Select All is available as its own "quick action" above each table
- [ ] Testing Checklist has been run and all tests pass
- [ ] README is updated, if necessary

**Tasks:**
_Create a list of granular, specific work items that must be completed to deliver the desired outcomes of this issue_ 

- [ ] _Task 1_
- [ ] _Task 2_
- [ ] _Task 3_
- [ ] Run Testing Checklist and confirm all tests pass

**Supporting Documentation:**
[HackMD notes](https://hackmd.io/3zR7vj7GQ7uidJmLR-eCYw?view#Follow-On-Stuff)

**Comment by ADPennington** [comment]
11/17 notes:
- caption should be number columns sorted re: open question 
- whatever should be read aloud should be in a "live region"

**Comment by valcollignon** [comment]
@ADPennington @abottoms-coder @lfrohlich - what is the priority of this ticket? It's been in "Next Sprint Backlog" since November 2021. :) 

Is this refined and ready to go, or should we move back to product backlog for some other time in the future?

**Comment by valcollignon** [comment]
> Is this refined and ready to go, or should we move back to product backlog for some other time in the future?

Same question for #1487, #1411, #1415

**Comment by abottoms-coder** [comment]
> > Is this refined and ready to go, or should we move back to product backlog for some other time in the future?
> 
> Same question for #1487, #1411, #1415

Moved to product backlog per backlog refinement 3.1.22.

**Comment by stevenino** [comment]
Fully refined. This is low priority because Django will only be used by a couple of people and there is higher priority work that is required to meet the release schedule goals.

**Comment by robgendron** [comment]
@andrew-jameson  is this still needed? If still relevant, should we look to move it into one of the roadmap epics?

**Comment by reitermb** [comment]
> @andrew-jameson is this still needed? If still relevant, should we look to move it into one of the roadmap epics?

This / other a11y tickets that are still open should all be up to date & still needed.

**Comment by lhuxraft** [comment]
May be addressed through a future Django upgrade

**Comment by reitermb** [comment]
Closing as not needed given decision to invest in react admin
